lesson5/main.js

```diff
@@ -1,15 +1,15 @@
 "use strict";
 
-const fragment = document.createDocumentFragment();
-const listItems = [
-  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
-  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" },
-];
 const insert = new Promise((resolve, reject) => {
+  const listItems = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" },
+  ];
   resolve(listItems);
 });
 
 function newList(data) {
```

Comment:
NITS:
This is also the matters about naming of `newList`.

Function names are become usually based on **"verb + object"**
In this case, the main purpose is to generate list elements, so I think "createListItem" or "createList" should be fine, right?

Currently, the word "new" is a little weak.

I will send you a reference link.

https://kitsune.blog/naming#%E9%96%A2%E6%95%B0%E5%90%8D%E3%81%AF%E5%8B%95%E8%A9%9E_%E7%9B%AE%E7%9A%84%E8%AA%9E%E3%81%AB%E3%81%99%E3%82%8B

Reply:
I didn't know function name based on verb + object!
I fixed them.
[2a6918e](https://github.com/yuka830/js-lessons/pull/7/commits/2a6918e296e5598b8792d317359f1a6b3b7c0dea)

Reply:
OK.
It's a small detail, but I think the word "new" is unnecessary since we know we are creating some new list.

With this to be Approve from me. 
Thank you for your hard work.

Reply:
You're right. It's clearly to make new list as we can see.I should have thought about it.
Thank you for your helping.